OSDOCS-3794: Restructured STS Prerequisites (#51465)
openshift/openshift-docs: modules/rosa-sts-aws-requirements-ocm.adoc

// Module included in the following assemblies:
//
// * rosa_getting_started_sts/rosa-sts-aws-prereqs.adoc

:_content-type: CONCEPT
[id="rosa-ocm-requirements_{context}"]
= Requirements for using {cluster-manager}

> The following sections are only required if you use {cluster-manager-url}. If you are only using your CLI tools, then you can disregard these requirements.

Reviewer: ```suggestion
The following sections describe requirements for {cluster-manager-url}. If you use the CLI tools exclusively, you can disregard the requirements.
```

Does this improve what you're trying to say, here?

> To use {cluster-manager}, you must link your AWS accounts. This linking concept is also known as account association.

Reviewer: Is this addition really a new assembly? Is account association a discrete user story?
